gen-app-builder/search-web-app/genappbuilder_utils.py

```diff
@@ -13,13 +13,38 @@
 # limitations under the License.
 
 """Generative AI App Builder Utilities"""
-from typing import List, Tuple
+from os.path import basename
+from typing import List, Optional, Tuple
 
-from google.cloud import discoveryengine
+from google.cloud import discoveryengine, discoveryengine_v1beta
```

Comment:
Is it dangerous to use objects/instances from `discoveryengine` and `discoveryengine_v1beta` at the same time? Might a user run into conflicts or compatibility issues? Maybe it's better to import only the `discoveryengine_v1beta` namespace and use everything from there? Or if not, then you can safely ignore this comment. 😁

Reply:
Valid point, I wasn't too sure about it. Wanted to encourage people to use the GA (v1) components whenever possible, but some features only support v1beta